src/main/java/at/overflow/bukkit/matrixbridge/BridgeService.java

```diff
@@ -83,6 +86,23 @@ public void run() {
                             if (msg.getBody() != null) {
                                 if (msg.getBody().startsWith("!")) {
                                     this.parseCommand(msg.getBody());
+				} else if (syncToken != null
+					   && msg.getBody().startsWith("/")
+					   && msg.getSender().getId().equals(properties.getAdmin())) {
```

Comment:
I think the "proper way" of handling commands in Matrix is the `!` prefix so it doesn't clash with build-in commands.
Is there a reason you choose `//`?

Reply:
I just wanted to stick to the minecraft-syntax for admin commands. Having `!players`would prevent interacting with any mod on the minecraft-side that introduces this command. Maybe `#` makes more sense, but it should be something different than `!` due to `!players`.

Reply:
I went with `$` for now.